src/components/BarChart/BarChart.jsx

```diff
@@ -237,6 +254,11 @@ const BarChart = createClass({
 			.domain([yAxisMin, yAxisMax])
 			.range([innerHeight, 0]);
 
+		const xFinalFormatter = xAxisFormatter;
+		const yFinalFormatter = yAxisFormatter
```

Comment:
That is more concise, and I'll remove `xFinalFormatter` since it's not doing anything. I can't move it into the destructure cause it relies on `yScale` which isn't available until this line.